src/Actions/Actions/LoadActionParts.cs

```diff
@@ -31,19 +34,25 @@ public class LoadActionParts
         /// </summary>
         public SnapshotMetaInfo MetaInfo { get; private set; }
 
+        /// <summary>
+        /// Custom UIA registrations
+        /// </summary>
+        IReadOnlyDictionary<int, CustomProperty> CustomProperties { get; }
+
         /// <summary>
         /// Constructor
         /// </summary>
         /// <param name="el">The element that was selected when the file was saved</param>
         /// <param name="bmp">Actual screenshot</param>
         /// <param name="synthesizedBmp">Synthesized ("yellow box") bitmap</param>
         /// <param name="settings">Metadata about the snapshot</param>
-        public LoadActionParts(A11yElement el, Bitmap bmp, Bitmap synthesizedBmp, SnapshotMetaInfo settings)
+        public LoadActionParts(A11yElement el, Bitmap bmp, Bitmap synthesizedBmp, SnapshotMetaInfo settings, IReadOnlyDictionary<int, CustomProperty> customProperties)
         {
             this.Element = el;
             this.Bmp = bmp;
             this.SynthesizedBmp = synthesizedBmp;
             this.MetaInfo = settings;
+            this.CustomProperties = customProperties;
```

Comment:
Do we need to store this? I'm not finding the code that reads it...

Reply:
We don't read it, just storing for consistency. Maybe we shouldn't do this?